ytfzf

```diff
@@ -973,6 +1000,11 @@ parse_long_opt () {
 
 		notification) parse_opt "N" ;;
 
+		add-pl) printf "Missing playlist name!"; exit ;;
+		add-pl=*) parse_opt "p" "${opt#*=}" ;;
+		play-pl) printf "Missing playlist name!"; exit ;;
```

Comment:
no need to worry about this check, they would get an error if they had used without `=` in the end